src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/DatePicker.js

```diff
@@ -121,17 +127,31 @@ export default class DatePicker extends React.Component<Props> {
         };
     };
 
-    getPlaceholder = (fieldOptions: {} => string) => {
-        const placeholderDate = fieldOptions.dateFormat ? fieldOptions.dateFormat : '';
-        const placeholderTime = fieldOptions.timeFormat ? moment.localeData().longDateFormat('LT') : '';
+    getDateFormat = (): string => {
+        let dateFormat = this.props.options.dateFormat;
 
-        if (!fieldOptions.timeFormat) {
-            return placeholderDate;
-        } else if (fieldOptions.dateFormat && fieldOptions.timeFormat) {
-            return placeholderDate + ' ' + placeholderTime;
+        if ((!dateFormat && dateFormat !== false) || dateFormat === true || (!dateFormat && !this.getTimeFormat())) {
+            dateFormat = moment.localeData().longDateFormat('L');
```

Comment:
Couldn't you just return here? Would allow to make the `dateFormat` above to be `const`. Or is it possible that this `longDateFormat` call returns a falsey value?